ern-container-gen/src/generators/android/hull/lib/src/main/java/com/walmartlabs/ern/container/ElectrodeReactContainer.java

```diff
@@ -245,12 +265,19 @@ private static void notifyReactNativeReadyListeners() {
         }
     }
 
+    @SuppressWarnings("unused")
     public static void resetReactNativeReadyListeners() {
         reactNativeReadyListeners.clear();
     }
 
     public interface ReactNativeReadyListener {
-            void onReactNativeReady();
+        void onReactNativeReady();
+    }
+
+    private static void throwNotInitializedStateException() {
```

Comment:
Could you rename this method to  something`throwIfNotInitialized()` ?
The current name of the method is misleading. The name of the method states that calling it will throw a `NotInitializedStateException`.

Reply:
sure, will update.